samples/weather_function/readme.md

```diff
@@ -0,0 +1,36 @@
+# Weather Function Sample
+
+This is a Serverless Function that fetches the current weather information. Part of the Function's output includes HTTP response headers, and the HTTP response body. This sample is designed to be deployed on OpenFaaS via a Docker image.
+
+This document assumes that you are using a PC running Debian (v9 or later), Ubuntu (v14.04 or later), or Linux Mint (v17 or later).
+
+
+# Building Docker Image
+
+It will be assumed that Docker is already installed.
+
+1. Clone the Kotlin Native Git repository: ```git clone https://github.com/JetBrains/kotlin-native ~/repos/kotlin-native```
```

Comment:
Is that path still correct?
Why should it be cloned to this path?

Reply:
The path provides a central place users can easily follow in the documentation.